KrotovaEY/lab_3/Source.cpp

```diff
@@ -0,0 +1,92 @@
+#include <iostream>
+#include <conio.h>
+using namespace std;
+
+int min(int *mass, int size)
+{
+	int min = mass[0];
+	int index;
+	for (int i = 0; i < size; i++)
+	{
+		if (mass[i] <= min)
+		{
+			min = mass[i];
+			index = i;
+		}
+	}
+	return index;
+}
+int main()
+{
+	const int size = 50;
+	char str[size] = " ";
+	setlocale(0, "");
+	cout << "Введите строку: ";
+	cin.getline(str, size);
+	int mas[size] = { 0 };
+	int  sizemas= 0;
+	for (int i = 0; i < size; i++)
+	{
+		if (str[i] != ' ' && str[i] != 0)
+		{
+			mas[sizemas] += int(str[i]);

```

Comment:
explicit cast to int is redundant here as char has implicit type conversion to int